cl/people_db/migrations/0043_load_tenn_judge_data.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2020-07-16 20:17
+
+from django.db import migrations
+import sys
+
+from cl.lib.migration_utils import load_migration_fixture
+
+fixture = "tenn_judges"
+
+
+def load_fixtures(apps, schema_editor):
+    if "test" not in sys.argv:
+        load_migration_fixture(apps, schema_editor, fixture, "people_db")
+
+
+def unload_fixture(apps, schema_editor):
+    Person = apps.get_model("people_db", "Person")
+    Position = apps.get_model("people_db", "Position")
+    new_pks = ["tennworkcompapp", "tennworkcompcl"]
+    for pk in new_pks:
+        for human in Position.objects.filter(court_id=pk):
+            pers = Person.objects.get(id=human.person_id)
+            human.delete()
+            pers.delete()
```

Comment:
I don't understand `human` vs. `person`, buuuut, perhaps it won't matter since I think we're moving away from this migration.

Reply:
Why should we move from this migration?

Reply:
See: https://github.com/freelawproject/courtlistener/pull/1341#discussion_r458422872